src/RawPOSIX/src/safeposix/syscalls/fs_calls.rs

```diff
@@ -130,6 +130,38 @@ impl Cage {
         ret
     }
 
+    //------------------------------------UNLINKAT SYSCALL------------------------------------
+    /*
+    *   unlinkat() will return 0 when success and -1 when fail 
```

Comment:
Do we cover all cases that unlinkat can have according to the man page?   If not, what is different?